src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -678,6 +678,7 @@ public virtual bool CanDetach()
         /// </summary>
         /// <returns>[Required] Task to track when this is complete</returns>
         abstract public Task EnableTargetAsyncOption();
+        abstract public Task<Results> ListTargetFeatures();
```

Comment:
/// <summary>
        /// Returns a Results object containing the set of features supported by GDB.
        /// </summary>
        /// <returns>Task that will contain the Results. The Results object is optional.</returns>

Reply:
Thanks for reviewing!

Reply:
Thanks for reviewing!